src/Symfony/Component/HttpFoundation/File/MimeType/MimeTypeExtensionGuesser.php

```diff
@@ -808,6 +808,6 @@ class MimeTypeExtensionGuesser implements ExtensionGuesserInterface
      */
     public function guess($mimeType)
     {
-        return isset($this->defaultExtensions[$mimeType]) ? $this->defaultExtensions[$mimeType] : null;
+        return isset($this->defaultExtensions[strtolower($mimeType)]) ? $this->defaultExtensions[strtolower($mimeType)] : null;
```

Comment:
perhaps store a lowercased `$lcMimeType` once, and maybe try isset() on the original mime type first (could be a legit optimization).

Reply:
@ro0NL I made the suggested optimizations, can you review them?

Reply:
i think we should add a test also :)